samba-dc/usr/local/sbin/expand-config

```diff
@@ -51,14 +51,36 @@ cat - >/etc/samba/smb.conf <<EOF
         server role = active directory domain controller
         workgroup = ${NBDOMAIN}
         log level = ${SAMBA_LOGLEVEL}
+
         acl_xattr:security_acl_name = user.NTACL
+        acl_xattr:ignore system acls = yes
+
+        template homedir = ${SAMBA_SHARES_DIR:?}/%U
```

Comment:
While this is good for new installation, are we sure it can't cause any conflict during migration?
Example: a share named after a user.